File: Java/1347.java
class Solution {
    public int minSteps(String s, String t) {
        int result = 0;
        int[] count = new int[26];
        
        for (int i=0; i<s.length(); i++){
            count[s.charAt(i) - 'a'] ++;
        }
        
        for (int j=0; j<t.length(); j++){
            if (count[t.charAt(j) - 'a'] -- <= 0)
                result ++;
        }
        
        return result;
    }
}

File: Python/1347.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        map_s = {}
        for char in s:
            if char in map_s:
                count = map_s.get(char)
                count = count+1
                map_s.update({char:count})
            else:
                map_s[char] = 1
                
        answer = 0
        for char in t:
            if char not in map_s:
                answer = answer+1
            else:
                count = map_s.get(char)-1
                map_s.update({char:count})
                if count<0:
                    answer = answer+1        
        return answer
